packages/mdc-tabs/package.json

```diff
@@ -4,6 +4,9 @@
   "description": "The Material Components for the web tabs component",
   "license": "MIT",
   "main": "dist/mdc.tabs.js",
+  "module": "index.js",
+  "types": "dist/mdc.tabs.d.ts",
```

Comment:
It looks like `md-tabs` is missing a `.npmignore` file

Reply:
Make sure all other packages have one too 😀

Reply:
I added it to tabs, removed it from selection-control (because #4442 deletes that), and checked everything else.

Reply:
Thanks!